tests/empty-expected.wat

```diff
@@ -0,0 +1,13 @@
+(module
```

Comment:
Is this file used anywhere?  I can't seem to see any references.

Reply:
Nope, removed!